### PR TITLE
Setup project structure

### PR DESCRIPTION
**Pull request by dolbilov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "imaybedepressed",
+  "version": "1.0.0",
+  "description": "Сайт для выявления депрессии и тревожности с помощью специальных тестов.",
```
Comment (HereHaveTheseFlowers) [inline package.json:4]: How about this:
```A website containing several tests that can help you better understand how you've been feeling recently.```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "url": "git+https://github.com/dolbilov/IMayBeDepressed.git"
+  },
+  "keywords": [],
+  "author": "Kirill Dolbilov & Sergey Kozlov",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/dolbilov/IMayBeDepressed/issues"
+  },
+  "homepage": "https://github.com/dolbilov/IMayBeDepressed#readme",
+  "devDependencies": {
```
Comment (HereHaveTheseFlowers) [inline package.json:23]: How about we remove all the `^` from the versions of dependencies? Im worried something might break in the future.

**Review by HereHaveTheseFlowers — APPROVED** [review]
Merge ready 🛰️
Just a few suggestions you might like.

File: package.json (hunk 1 above)
Comment (dolbilov) [reply package.json:4]: We make site for Russian people, aren't we?

Comment (dolbilov) [reply package.json:4]: Changed.

File: package.json (hunk 2 above)
Comment (dolbilov) [reply package.json:23]: Deleted. Thank you.

**Pull request merged**